client/src/main/java/com/arcbees/beeshop/client/application/ApplicationView.java

```diff
@@ -16,17 +16,25 @@
 
 package com.arcbees.beeshop.client.application;
 
+import static com.google.gwt.query.client.GQuery.$;
```

Comment:
We have to agree on the code style, or else we'll keep getting "static imports moved at top". I think it goes after in Arcbees' code style.